mitxgraders/formulagrader/matrixgrader.py

```diff
@@ -59,8 +60,23 @@ class MatrixGrader(FormulaGrader):
                         received objects is revealed.
 
         suppress_matrix_messages (bool): If True, suppresses all matrix-related
-            error messages from being displayed. Overrides shape_errors=True and
+            erro2r messages from being displayed. Overrides shape_errors=True and
             is_raised=True. Defaults to False.
+
+        Additionally, the following three configuration keys will be passed to
+        the default comparer and be used to specify partial credit options.
+
+        entry_partial_credit ('proportional'|number): Determines how partial credit
+            is awarded. If set to 'proportional', then credit is proportional to
+            the number of correct matrix entries. If a numeric value is provided,
+            this flat rate of partial credit is provided as long as some but
+            not all entries are correct.
+
+            Default is the numeric value 0 (no partial credit).
+        entry_partial_msg (str): A text string message shown when partial credit
+            is awarded. The string may optionally contain the formatting key {error_indices},
+            which will be replaced with the indices of the incorrect matrix entries.
+        transform (None | function): same as EqualityComparer, defaults to None
```

Comment:
I think you meant to leave off transform? At least, you did in the code below. I think it makes sense to leave off `transform`; anybody wanting to use that key should set up the comparer properly. This means that there's only two configuration keys (see above - `Additionally, the following three...`).